src/TokenLogicFunding_20230919/TokenLogicFunding_20230919.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+
+/*
+   _      ΞΞΞΞ      _
+  /_;-.__ / _\  _.-;_\
+     `-._`'`_/'`.-'
+         `\   /`
+          |  /
+         /-.(
+         \_._\
+          \ \`;
+           > |/
+          / //
+          |//
+          \(\
+           ``
+     defijesus.eth
+*/
+
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveSwapper} from 'aave-helpers/swaps/AaveSwapper.sol';
+import {AaveMisc} from 'aave-address-book/AaveMisc.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+
+/**
+ * @dev (1) Swap aEthDAI to GHO; (2) Create stream to Tokenlogic
+ * @author defijesus.eth - TokenLogic
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x53728c0416a9063bf833f90c3b3169fa4387e66549d5eb2b7ed2747bfe7c23fc
+ * - Discussion: https://governance.aave.com/t/arfc-treasury-management-replace-agd-s-dai-allowance-with-gho-allowance/14631
+ */
+contract TokenLogicFunding_20230919 is IProposalGenericExecutor {
+
+  struct Asset {
+    address underlying;
+    address aToken;
+    address oracle;
+    uint256 amount;
+  }
+  
+  function execute() external {
+    address MILKMAN = 0x11C76AD590ABDFFCD980afEC9ad951B160F02797;
+    address PRICE_CHECKER = 0xe80a1C615F75AFF7Ed8F08c9F21f9d00982D666c;
+    AaveSwapper SWAPPER = AaveSwapper(AaveMisc.AAVE_SWAPPER_ETHEREUM);
+
+    Asset memory DAI = Asset({
+      underlying: AaveV3EthereumAssets.DAI_UNDERLYING,
+      aToken: AaveV3EthereumAssets.DAI_A_TOKEN,
+      oracle: 0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9,
+      amount: 350_000 * 1e18
+    });
+
+    Asset memory GHO = Asset({
+      underlying: AaveV3EthereumAssets.GHO_UNDERLYING,
+      aToken: address(0), // not used
+      oracle: 0x3f12643D3f6f874d39C2a4c9f2Cd6f2DbAC877FC,
+      amount: 350_000 * 1e18
+    });
+
+    /// 1. withdraw DAI & swap to GHO
+
+    AaveV3Ethereum.COLLECTOR.transfer(DAI.aToken, address(this), DAI.amount);
```

Comment:
[Collector](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c) has just 60k of aDAI, so transfer of 350_000 will fail

Reply:
ah, sorry, etherscan is not showing v3